lib/Controller/AdminController.php

```diff
@@ -8,35 +8,35 @@
 use OCA\Recognize\Classifiers\Images\ImagenetClassifier;
 use OCA\Recognize\Classifiers\Images\LandmarksClassifier;
 use OCA\Recognize\Classifiers\Video\MovinetClassifier;
+use OCA\Recognize\Db\FaceClusterMapper;
+use OCA\Recognize\Db\FaceDetectionMapper;
 use OCA\Recognize\Service\QueueService;
 use OCA\Recognize\Service\TagManager;
 use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http;
 use OCP\AppFramework\Http\JSONResponse;
 use OCP\BackgroundJob\IJobList;
+use OCP\DB\Exception;
 use OCP\IConfig;
+use OCP\IDBConnection;
```

Comment:
```suggestion
```

Please add codesniffer and psalm to this application :-P
You can copy files from https://github.com/nextcloud/.github/tree/master/workflow-templates

Reply:
My recommendations would be:
* command-compile.yml
* command-rebase.yml
* dependabot-approve-merge.yml
* fixup.yml
* lint-eslint.yml
* lint-info-xml.yml
* lint-php-cs.yml
* lint-php.yml
* lint-stylelint.yml
* node.yml
* psalm.yml
* update-nextcloud-ocp.yml